### PR TITLE
[Items] CAES-954: Add site logo

### PR DESCRIPTION
**Pull request by sa715**
Signed-off-by: Sergey Alekseyev <sa@4xxi.com>

### CODE REVIEW

File: packages/components/ItemIcon/ItemIcon.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+const ItemIcon = ({ url }) =>
+  <img src={`https://www.google.com/s2/favicons?domain=${url}`} alt="" />;
+
+export default ItemIcon;
```
Comment (cu4xxi) [inline packages/components/ItemIcon/ItemIcon.js:6]: let's use named imports and exports.
Default exports are deprecated and rewritten now

File: packages/components/List/Item/styles.js [hunk 2]
```diff
@@ -133,7 +136,7 @@ export const Row = styled.div`
   }
   
   ${IconWrapper} {
-    display: ${({ isMultiItem }) => (isMultiItem ? 'none' : 'inline-block')};
+    display: ${({ isMultiItem }) => (isMultiItem ? 'none' : 'flex')};
```
Comment (cu4xxi) [inline packages/components/List/Item/styles.js:139]: maybe, inline-flex?
// i'm not sure, but seems that inline-block was here for a reason
// nitpick

**Review by cu4xxi — APPROVED** [review]

File: packages/components/ItemIcon/ItemIcon.js (hunk 1 above)
Comment (sa715) [reply packages/components/ItemIcon/ItemIcon.js:6]: Done

File: packages/components/List/Item/styles.js (hunk 2 above)
Comment (sa715) [reply packages/components/List/Item/styles.js:139]: IconWrapper is a span element and `inline-block` was used in opposite of `none` for display property. So now `flex` works fine here

**Review by rtatarinov4xxi — APPROVED** [review]

**Pull request merged**